components/event-publisher-proxy/pkg/sender/jetstream.go

```diff
@@ -39,6 +39,11 @@ func NewJetstreamMessageSender(ctx context.Context, connection *nats.Conn, envCf
 	return &JetstreamMessageSender{ctx: ctx, connection: connection, envCfg: envCfg, logger: logger}
 }
 
+// URL returns the URL of the Sender's connection.
+func (s *JetstreamMessageSender) URL() string {
+	return s.connection.ConnectedUrl()
```

Comment:
have you tested this? does it return the correct svc url?

Reply:
yes, here is an example metric that results out of this:
```
event_publish_to_messaging_server_latency_bucket{destination_service="eventing-nats.kyma-system.svc.cluster.local",response_code="204",le="+Inf"} 340
```